utilities/exceptions.py

```
@@ -67,6 +67,15 @@ class StorageCheckupConditionTimeoutExpiredError(Exception):
     pass
 
 
+class DataVolumeConditionMessageNotFoundError(Exception):
+    def __init__(self, dv_name: str, expected_message: str):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)